src/utils/getIconClass.ts

```diff
@@ -0,0 +1,15 @@
+export const getIconClass = (
+  field: string,
+  sortField: string,
+  isReversed: string,
+) => {
+  if (sortField === field) {
+    if (isReversed) {
+      return 'fa-sort-down';
+    }
+
+    return 'fa-sort-up';
+  }
+
+  return 'fa-sort';
+};
```

Comment:
```suggestion
  if (sortField !== field) {
    return 'fa-sort';
  }
  
  if (isReversed) {
    return 'fa-sort-down';
  }

   return 'fa-sort-up';
};
```

try to avoid nesting